web/models/release_metadata.ex

```diff
@@ -12,5 +12,15 @@ defmodule HexWeb.ReleaseMetadata do
     |> validate_required(~w(app build_tools)a)
     |> validate_list_required(:build_tools)
     |> update_change(:build_tools, &Enum.uniq/1)
+    |> validate_elixir()
+  end
+
+  defp validate_elixir(changeset) do
```

Comment:
Maybe make this more generic and add it to `validations.ex`?

`validate_requirement(changeset, field)`